src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -167,8 +167,9 @@ def test_sample_numpoints():
     _a = 1.0 * u.deg  # Unused angle
     _body = Sun  # Unused body
     ss = Orbit.from_classical(_body, _d, _, _a, _a, _a, _a)
+    times, positions = ss.sample(values=50)
 
-    assert len(ss.sample(values=50)) == 50
+    assert len(positions) == 50 and len(times) == 50
```

Comment:
What about `len(positions) == len(times) == 50`? :)

Reply:
Yep, that is indeed a better way to check this. Gotta fix this real quick.